ls1intum/Artemis
# Add breadcrumb navigation

### Checklist

### Motivation and Context
See #2097.

### Description
Added bread crumbs to the navigation bar in the top. This is currently restricted to users above the "normal" user (tutors, instructors, admins) and to the routes `<base-url>/#/admin` and `<base-url>/#/course-management`. Better labelling (e.g. a course name instead of the id) will be added in a future PR.

### Steps for Testing

1. Open pages matching the routes `<base-url>/#/admin` or `<base-url>/#/course-management` and check the crumbs are correctly displayed and work
2. Check that users without extra permissions don't get crumbs

Note: Only test that the links work the same as if you typed it in the browser. Not all crumbs make sense right now since some of them are redirects. (Especially the `view` pages need to go. That is part of the reason we don't show it to normal users yet.)

### Screenshots
Examples:
![grafik](https://user-images.githubusercontent.com/72132281/97303715-0af74580-185b-11eb-94bc-7222f343e9df.png)
![grafik](https://user-images.githubusercontent.com/72132281/97506487-5274f300-197b-11eb-86ec-10387213103c.png)
